.github/ISSUE_TEMPLATE/BugReport.yml

```
@@ -0,0 +1,107 @@
+name: Bug Report
+description: Is the SDK not working as expected? Help us improve by submitting a bug report.
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Ensure you go through our [troubleshooting](https://docs.datadoghq.com/real_user_monitoring/mobile_and_tv_monitoring/troubleshooting/#react-native) page before creating a new issue. 
+        Before getting started, if the problem is urgent or easier to investigate with access to your organization's data please use our [official support channel](https://www.datadoghq.com/support/).
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe the bug
+      description: Provide a clear and concise description of what the bug is.
+    validations:
+      required: true
+  - type: textarea
+    id: reproduction
+    attributes:
+      label: Reproduction steps
+      description: |
+        Provide a self-contained piece of code demonstrating the bug. 
+        For a more complex setup consider creating a small app that showcases the problem. 
+        **Note** - Avoid sharing any business logic, credentials or tokens.
+    validations:
+      required: true
+  - type: textarea
+    id: logs
+    attributes:
+      label: SDK logs
+      description: |
+        Please provide console logs before, during and after the bug occurs. 
```

The important information for troubleshooting is often in the native logs, rarely in the JS console logs. 
So I'd suggest to mention them explicitely, with a link explaining how to get them. Do you think that's a good idea or too much to ask for?

```suggestion
        Please provide javascript and native logs from before, during and after the bug occurs. Refer to [our troubleshooting page](https://docs.datadoghq.com/real_user_monitoring/mobile_and_tv_monitoring/troubleshooting/#3-review-native-logs) on how to collect native logs.
```

---

No, it's perfectly reasonable and likely something we'd end up doing via additional comments.
Good catch.